lib/menu.js

```diff
@@ -1,5 +1,7 @@
 const { LitElement, html, css } = require('lit-element')
-const config = require('../config.json')
+const req = require('sync-request')
```

Comment:
I noticed now that this depends on a bunch of stuff

It's not really needed to do this sync. Could you instead do this async with fetch ?